storage/redis/client.go

```diff
@@ -49,7 +48,7 @@ func NewClient(address, password string, db int) (*Client, error) {
 
 // NewClientFrom returns a configured Client instance from a redis address, verifying a successful connection to redis
 func NewClientFrom(address string) (*Client, error) {
-	redisurl, err := utils.ParseURL(address)
+	redisurl, err := url.Parse(address)
```

Comment:
url.Parse doesn't handle boltdb properly. e.g. `bolt://C:\example.txt`.. although it's probably fine here, since the address should only be for redis.